src/util/tracking.ts

```diff
@@ -64,7 +66,7 @@ export async function logEvent(event: TrackingEventName, values: TrackingValues
  * Send a raw event to Firebase.
  */
 async function logToFirebase(name: TrackingEventName, values: TrackingValues) {
-  const { accountDate, currencyCode, dollarValue, installerId, pluginId, error } = values
+  const { accountDate, currencyCode, dollarValue, installerId, pluginId, error, referralId } = values
```

Comment:
Looks like we were already have the installerId as a param. Can you just use that without adding referralId?